apps/pm-user/src/components/molecules/bigRecommendationCard/BigRecommendationCard.js

```diff
@@ -0,0 +1,206 @@
+import { useState } from 'react';
+import { showNotification } from '@pm/pm-ui';
+import PropTypes from 'prop-types';
+import { useHistory } from 'react-router-dom';
+import styles from './bigRecommendationCard.module.scss';
+import {
+  Button,
+  Carousel,
+  EyeOutlined,
+  HeartOutlined,
+  Modal,
+  PhoneOutlined,
+  ProfileOutlined,
+} from '../../atoms/';
+
+const BigRecommendationCard = ({
```

Comment:
The name "BigRecommendationCar" does not suit it.
because this card is showing other users' information and will be used on other pages also like on the "Matches page" for shortlisted profiles, Interest sent, Interest received, etc.

Reply:
Ok. Renaming it to UserInfoCard

Reply:
Resolved all issues.

Reply:
This is the updated link.

![image](https://user-images.githubusercontent.com/48716076/185791145-03c031f7-f63d-484f-ba66-84f7ecfd0b9c.png)